docker-entrypoint.sh

```diff
@@ -73,6 +76,17 @@ if [[ "$1" == "lnd" || "$1" == "lncli" ]]; then
         echo "externalip=$HIDDENSERVICE_ONION added to $LND_DATA/lnd.conf"
     fi
 
+    # if it is legacy installation, then trigger warning and add noseedbackup=1 to config if needed
+    WALLET_FILE="$LND_DATA/data/chain/$NETWORK/$ENV/wallet.db"
```

Comment:
Are these constant? Can the wallet location be set through lnd.conf or `$LND_EXTRA_ARGS`?

Reply:
Well, for our case it definitely won't matter as we are controlling the variables.

But yeah, if it's possible to influence wallet location through lnd.conf (which is where $LND_EXTRA_ARGS end up in this case) - and someone else will depend on our Docker container + wants to change wallet path - then we would need to parse lnd.conf and figure out wallet path in order to init this variable.

For now I'll just resolve conversation... if you, good person from the future, come up over this conversation and need this addressed, open new issue.